### ISSUE TITLE
.designspace <dimension> uservalue attribute not supported?

### ISSUE THREAD

**Issue by justvanrossum** [googlefonts/fontc#813]
fontc fails if the instances in my designspace file use the `uservalue` attribute in their `<dimension>` elements, instead of `xvalue`.

Is this a known limitation?

https://fonttools.readthedocs.io/en/latest/designspaceLib/xml.html#dimension-element

**Comment by anthrotype** [comment]
> Is this a known limitation?

yes, sorry I thought we had an issue tracking DSv5 support but turns out we didn't.

https://github.com/googlefonts/fontc/issues/837
